src/odemis/acq/acqmng.py

```diff
@@ -83,6 +84,189 @@ def acquire(streams, settings_obs=None):
     return future
 
 
+def acquireZStack(streams, zlevels, settings_obs=None):
+    """
+    The acquisition manager of a zstack
+    streams (list of Stream): the streams to be acquired
+    zlevels (dict Stream -> list of floats): a dictionary containing the streams and the
+    corresponding lists of actuator z positions
+    settings_obs (SettingsObserver):
+    return (ProgressiveFuture): the future that will be executing the task
+    """
+    # TODO settings observer
```

Comment:
Do you mean create settings observer? Test settings observer? Update settings of the settings observer? One verb would help the next reader of this code.